### PR TITLE
Gui set geometry

### PR DESCRIPTION
**Pull request by Rebecca-Rumney**
Set Geometry function is now functional. Informally tested by just building the window and seeing whether it worked.

### CODE REVIEW

**Comment by Rebecca-Rumney** [comment]
I also changed the __init__ of the class to have some variables required by the function

File: PKPD/gui/abstractGui.py [hunk 1]
```diff
@@ -1,46 +1,39 @@
+from PyQt5.QtWidgets import QMainWindow
 from abc import ABC, abstractmethod
```
Comment (DavAug) [inline PKPD/gui/abstractGui.py:3]: abc is no longer necessary. Please remove

**Comment by simonmarchant** [comment]
Everything looks good to me Rebecca! Obviously we probably don't want the under-construction stuff once it gets to master but that's not yet. If you merge in the setup.py then it'll pass the travis tests, which we may as well get before merging.

**Review by barnums — COMMENTED** [review]
All looks good to me!

File: PKPD/gui/mainWindow.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        self.bottom_bar = None
+        self.testUI()
+
+
+    def _set_geometry(self):
+        """Function which sets the layout of the main window"""
+        self.setMenuBar(self.top_bar)
+        self.setCentralWidget(self.tab_view)
+        self.setStatusBar(self.bottom_bar)
+
```
Comment (DavAug) [inline PKPD/gui/mainWindow.py:23]: testing separate from class

File: PKPD/gui/mainWindow.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        self.top_bar = QMenuBar()
+        self.top_bar.addMenu('File')
+        self.tab_view = QPlainTextEdit()
+        self.bottom_bar = QStatusBar()
+        version_number = QLabel('Version: 0.0.0')
+        self.bottom_bar.addWidget(version_number)
+        self._set_geometry()
+
+        self.setWindowTitle('PKPD')
+        self.showMaximized()
```
Comment (DavAug) [inline PKPD/gui/mainWindow.py:33]: show probably also better not automatically executed within a test

Comment (DavAug) [reply PKPD/gui/mainWindow.py:33]: would make automated testing with travis difficult

**Review by DavAug — CHANGES_REQUESTED** [review]

**Comment by simonmarchant** [comment]
All changes from set geometry are incorporated into #22 so closing this.

**Pull request closed without merging**